ci/check-cherry-picks.sh

```diff
@@ -75,23 +101,27 @@ while read -r new_commit_sha ; do
         '
 
         if $range_diff_common --no-color 2> /dev/null | grep -E '^ {4}[+-]{2}' > /dev/null ; then
-          if [ "$GITHUB_ACTIONS" = 'true' ] ; then
-            echo ::endgroup::
-            echo -n "::warning ::"
-          else
-            echo -n "  ⚠ "
-          fi
-          echo "Difference between $new_commit_sha and original $original_commit_sha may warrant inspection:"
+          log success "$original_commit_sha present in branch $picked_branch"
+          endgroup
+          log warning "Difference between $new_commit_sha and original $original_commit_sha may warrant inspection."
 
-          $range_diff_common --color
+          # First line contains commit SHAs, which we already printed.
+          $range_diff_common --color | tail -n +2
```

Comment:
There is a limit how much text is allowed in comments.
You can look it up here: https://github.com/Mic92/nixpkgs-review/pull/491/files

Reply:
> You can look it up here: [Mic92/nixpkgs-review#491 (files)](https://github.com/Mic92/nixpkgs-review/pull/491/files)

> ```py
> # https://github.com/orgs/community/discussions/27190
> MAX_GITHUB_COMMENT_LENGTH = 65536
> ```

> Should we handle this here or let it fail instead?

Can we use `wc` to test whether to include the diff in the comment or not, but still print it to the CI logs regardless?

It's probably not the end of the world if it fails, since that'd make the CI job fail and also block the PR. At least until requiring status checks befor merge.

But it's not the ideal DX.

> So I'd say, we let it fail for now and see where that goes?

I think it's reasonable to be left for followup work, maybe highlight it as a TODO comment and/or a note on the PR description?

Reply:
For normal code changes I don't expect that we would hit the limit. However we also have sometimes large lock file changes that may fall into this category.